src/math.rs

```diff
@@ -15,17 +16,7 @@ macro_rules! no_mangle {
     }
 }
 
-#[cfg(any(
-    all(
-        target_family = "wasm",
-        target_os = "unknown",
-        not(target_env = "wasi")
-    ),
-    target_os = "xous",
-    target_os = "uefi",
-    all(target_arch = "xtensa", target_os = "none"),
-    all(target_vendor = "fortanix", target_env = "sgx")
-))]
+#[cfg(not(target_os = "windows"))]
```

Comment:
```suggestion
#[cfg_attr(all(not(windows), not(target_vendor = "apple")), weak)]
```

Reply:
We do have this already inside `no_mangle`, so probably just remove the `cfg` completely is a better choice

Reply:
No, we shouldn't provide the math intrinsics at all on windows/apple targets since weak linkage doesn't work properly. On those target we should always be getting these symbols from libc. The only exception is `lgamma_r` and `lgammaf_r` which are missing on MSVC targets.

Reply:
then you probably mean to use `cfg` and not `cfg_attr`